packages/core/src/routes/interaction/verifications/mfa-verification.ts

```diff
@@ -222,16 +222,31 @@ export const validateBindMfaBackupCode = async (
 
   if (
     !factors.includes(MfaFactor.BackupCode) ||
-    bindMfas.length === 0 ||
     bindMfas.some(({ type }) => type === MfaFactor.BackupCode)
   ) {
     return interaction;
   }
 
+  // Skip check if there is no new normal MFA
+  if (
+    event === InteractionEvent.Register &&
+    !bindMfas.some(({ type }) => type !== MfaFactor.BackupCode)
+  ) {
+    return interaction;
+  }
```

Comment:
move this blow `if (event === InteractionEvent.SignI)` and only check 

```
if (!bindMfas.find(({ type }) => type !== MfaFactor.BackupCode)) {
  return interaction;
}
```

Reply:
updated